src/main/java/trivia/roll/OddRole.java

```diff
@@ -0,0 +1,28 @@
+package trivia.roll;
+
+import trivia.Board;
+import trivia.player.Players;
+import trivia.question.Questions;
+
+import static trivia.roll.Roll.isOdd;
+
+class OddRole implements Roll {
+
+  private final BaseRoll baseRoll;
+
+  OddRole(int value) {
+    if (!isOdd(value)) {
+      throw new IllegalArgumentException("value must be odd");
+    }
+    this.baseRoll = new BaseRoll(value);
+  }
+
+  @Override
+  public void applyTo(Players players, Board board, Questions questions) {
```

Comment:
With this approach we are coupling business logic for getting out of penalty box with the roll. Roll shouldn't be responsible for moving in/out of Player from penalty box. This is accidental coupling. 

Moreover, it is easier to follow game flow expressed by the if/else statements than abstracted in classes that doesn;t really exist as natural objects. 

example of a flow for roll method:

```
      if (jail.hasImprisoned(players.getCurrentPlayer())) {
         jail.tryToGetOut(players.getCurrentPlayer(), roll);
         if (jail.hasImprisoned(players.getCurrentPlayer())) {
            return;
         }
      }
      //move
     //ask question
```